readthedocs/oauth/models.py

```diff
@@ -55,49 +44,73 @@ class RemoteOrganization(models.Model):
         null=True,
         blank=True,
     )
-
-    json = models.TextField(_('Serialized API response'))
+    remote_id = models.CharField(
+        db_index=True,
+        max_length=128
+    )
+    vcs_provider = models.CharField(
+        _('VCS provider'),
+        choices=VCS_PROVIDER_CHOICES,
+        max_length=32
+    )
 
     objects = RemoteOrganizationQuerySet.as_manager()
 
+    class Meta:
+        ordering = ['name']
+        unique_together = (('remote_id', 'vcs_provider'),)
```

Comment:
This syntax feels weird. Sounds like we can just do a single list: https://docs.djangoproject.com/en/2.2/ref/models/options/#django.db.models.Options.unique_together

```suggestion
        unique_together = ('remote_id', 'vcs_provider')
```